sysconfig/cloudinit.go

```diff
@@ -76,21 +80,50 @@ func installCloudInitCfg(src, targetdir string) error {
 	return nil
 }
 
-// TODO:UC20: - allow cloud.conf coming from the gadget
-//            - think about if/what cloud-init means on "secured" models
+// installUnifiedCloudInitCfg installs a single cloud-init config file to the
+// cloud config dir as "ubuntu-core-cloud_91.cfg".
```

Comment:
comment needs updating.

nitpick: from the description is not clear that this is specific to copying the config from the gadget. Either it needs to be made more specific or something more provided from outside?

Reply:
mmm yeah I hadn't thought about that yes, I think right now let's just rename the function here and we can refactor to install a single unified config that comes from what's on ubuntu-seed combined together later, that work is part of the maas grade: signed stuff which I haven't started on and would rather not get bogged down with right now